app/views/posts/show.html.erb

```diff
@@ -0,0 +1,10 @@
+<p id="notice"><%= notice %></p>
+
+<p>
+  <strong>Post:</strong>
+  <%= @post.post %>
+</p>
+<% if signed_in? %>
+<%= link_to 'Edit', edit_post_path(@post) %> |
+<% end %>
```

Comment:
* [ ] **_There should be NO application logic in the views (no `Post.new`, `Comment.create`, `current_user.friends.find_by(...)`...)._**
* [ ] **_View-related logic can be encapsulated in View helpers (app/helpers/\*_helper.rb)_**
* [ ] **_There should be absolutely NO logic in views._**

Reply:
We don't know how to render an HTML file in a normal rb file. Our logic as far as we understand is not an application logic but is a view logic. Still, however, if it is not the right way, can you help us and give us some examples of how to do it?

Reply:
We've deleted this edit part.